addon/components/one-way-input.js

```diff
@@ -86,12 +86,16 @@ const OneWayInputComponent = Component.extend({
   },
 
   init() {
-    this._super(...arguments);
```

Comment:
Yes, that was fixed with https://github.com/emberjs/ember.js/pull/12260 and was backported to 1.13.9 and 2.0.1.  What version were you testing with?